### DISCUSSION TITLE
v3 - Disable Delve link

### DISCUSSION THREAD

**Discussion by Heavm00** [microsoft-search/pnp-modern-search#3387] [General]
Hi everyone,

is there a simple way to deactivate the link to delve profile?

Thanks,
Evelyn

**Comment by kasperbolarsen** [comment]
Hi,
Where do you see a link to Delve?

**Reply by Heavm00** [reply]
Hi, when using search result v3 + people search results the Title (= lastname, firstname) gets an link to the delve profile. I would like to disable the link.

**Reply by kasperbolarsen** [reply]
V3 was deprecated about two years ago,  and is no longer supported. 
Your options are either to tweak the v3 code and build your own version or to switch to v4.
As v4 is very different from v3 on a code level it is not possible to upgrade from v3 to v4.
Perhaps this article can aid our decision,  https://sympmarc.com/2021/04/06/upgrading-the-pnp-modern-search-web-parts-from-v3-to-v4-where-are-they/

**Reply by Heavm00** [reply]
Hi @kasperbolarsen, thanks I know that all. I thought there would be an option to force to remove the link by JavaScript or CSS.
In that I case I'll update to v4.
